.gitlab-ci.yml

```diff
@@ -150,8 +150,6 @@ test:smoketests:linux:
     - ./mender-artifact-linux read test-rfs.mender
     - ./mender-artifact-linux validate test-rfs.mender
     - ./tests/test_compressions/test_supported_compressions.sh
-    # QA-507: lock mender-artifact to OpenSSL 1.1
-    - ldd ./mender-artifact-linux | grep libssl.so.1.1
```

Comment:
We should not break compatibility this way. We need to plan it a bit. As discussed in person, we could offer two flavors in our [Downloads page](https://docs.mender.io/downloads#mender-artifact): one statically linked without pkcs11 and one dynamically linked for openssl3.

As this starts to be bigger than a nice to have PR, can we create a task for it? It involves the work here, plus having some sort of subdirs structure in the S3 bucket, and `mender-docs`.